elogind: mention polkit for loginctl {poweroff,reboot} (#694)
void-linux/void-docs: src/config/power-management.md

`elogind` to `ignore` ACPI events in
[logind.conf(5)](https://man.voidlinux.org/logind.conf.5). There are several
configuration options, all starting with the keyword `Handle`, that should be

> set to `ignore` to avoid interfering with `acpid`. In order to be able to use
> `loginctl {poweroff,reboot}`as a normal user it is needed to install the polkit
> package.

Reviewer: I think this is better added in a separate paragraph:

```suggestion
set to `ignore` to avoid interfering with `acpid`.

In order to be able to use `loginctl poweroff` and `loginctl reboot` as a normal user, install the `polkit` package.
```

Reviewer: Also, I think I recall that `polkit` is actually required to run something like Sway at all?

Author: Thanks, sorry for the delay totally forgotten this.

> Also, I think I recall that `polkit` is actually required to run something like Sway at all?

I recall that also, not 100%, but I think that without polkit package installed and service running, it will not launch. (doesn't require a polkit client).

configuration options, all starting with the keyword `Handle`, that should be
set to `ignore` to avoid interfering with `acpid`.

> In order to be able to use `loginctl {poweroff,reboot}`as a normal user it is
> needed to install the `polkit` package.

Reviewer: ```suggestion
In order to be able to use `loginctl poweroff` and `loginctl reboot` as a
normal user it is needed to install the `polkit` package.
```

> normal user it is needed to install the `polkit` package.

Reviewer 2: maybe

> To run `loginctl poweroff` and `loginctl reboot` without root privileges, `polkit` must be installed.

Author: Sounds okay to me, that changed.
